src/Servers/HttpSys/src/MessagePump.cs

```diff
@@ -172,7 +173,7 @@ private async void ProcessRequestsWorker()
                 }
                 try
                 {
-                    Task ignored = Task.Factory.StartNew(_processRequest, requestContext);
+                    ThreadPool.UnsafeQueueUserWorkItem(_processRequest, requestContext, preferLocal: false);
```

Comment:
👍

Reply:
Nah, we need to implement IThreadPoolWorkItem on ApplicationWrapper to make it 0 alloc 😄

Reply:
Even better. Stopping the FIFO dispatching by itself is really nice though.

Reply:
Added the IThreadPoolWorkItem change https://github.com/aspnet/AspNetCore/pull/10283/commits/073ee5f55373e1bfa41bfa9349e76fe7d3169a39